### DISCUSSION TITLE
sunbird-rc-core - different password for each user

### DISCUSSION THREAD

**Discussion by maheshkumargangula** [Sunbird-RC/community#118] [Issues]
The registry microservice creates the user in Keycloak by using the default password configuration from the application.yml file. Having the same password for every user with the invite or registration process is a security concern.

Is there any way to generate a different password for each user when it is created?

Cc: @vrayulu

**Comment by dileepbapat** [comment]
In the development version, mock service sets the predetermined password just to ease the testing and getting started process. Once actual OTP plugin is configured / enabled it will send random otp.
